settings.gradle

```diff
@@ -8,3 +8,5 @@ rootProject.name = 'opensearch-knn'
 include ":qa"
 include ":qa:rolling-upgrade"
 include ":qa:restart-upgrade"
+
+startParameter.excludedTaskNames=["publishPluginZipPublicationToMavenLocal"]
```

Comment:
Hey we dont need to exclude this task as in build.sh we dont call `publishAllPublicationsToStagingRepository` 
https://github.com/opensearch-project/opensearch-plugins/blob/main/BUILDING.md#opensearchpluginzip (Point 7)

Reply:
ack